packages/data-provider/test/utils/formatUtils.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+
+import { isArrayWithValues, isEmptyString, isValidIdNumber, uniqueCharacters } from '../../src/utils/formatUtils.js';
+
+describe('Format Utils tests', () => {
+	describe('Validate if input is a valid ID number', () => {
+		it('should return true if input is a valid number', () => {
+			const validNumber = 1;
+			const response = isValidIdNumber(validNumber);
+			expect(response).to.be.true;
+		});
+
+		it('should return false if input is a not a valid number', () => {
+			const invalidNumber = 'one';
```

Comment:
Are there other edge cases to test? empty values, numeric strings like `'1'`, `Infinity`/`NaN`, null/undefined, negative numbers, `0`, numbers beyond max int...